projects/plugins/boost/app/lib/minify/functions-helpers.php

```diff
@@ -260,7 +260,15 @@ function jetpack_boost_page_optimize_cache_bust_mtime( $path, $siteurl ) {
 }
 
 /**
- * Detects requests within the `/_static/` directory, and serves minified content.
+ * Get the URL prefix for static minify/concat resources. Defaults to /jb_static/, but can be
+ * overridden by defining JETPACK_BOOST_STATIC_PREFIX. Please include leading and trailing slashes.
```

Comment:
You're right. I was avoiding it because in my head, we didn't want to add overhead for this check -- but the overhead would be negligible and I was optimizing the wrong thing.

Changed in e5aea543be.